kit/transport/http/error_handler.go

```diff
@@ -28,11 +28,7 @@ func (h ErrorHandler) HandleHTTPError(ctx context.Context, err error, w http.Res
 
 	code := errors2.ErrorCode(err)
 	var msg string
-	if err, ok := err.(*errors2.Error); ok {
-		msg = err.Error()
-	} else {
-		msg = "An internal error has occurred"
-	}
+	msg = err.Error()
```

Comment:
Do we want to keep `An internal error has occurred` as a prefix for the returned message? Or is that just noise?

Reply:
Unless we want to tag it as an error not in the expected format.

Reply:
Something like `unknown error type:`